refactor/config_data/config.json

```diff
@@ -152,18 +115,64 @@
     "description": "The breeding strategy. Can be 'maternal', 'quantitative' or 'discrete'",
     "value": "discrete"
   },
-  "pheno_resistance": {
-    "description": "Lookup table to translate between genotype.",
+  "geno_mutation_rate": {
+    "description": "Mutation rate, as in number of individual mutation events that can happen out of the number of offsprings. For example, 1e-5 is one mutation every 10,000 extruded eggs",
+    "value": 1e-5
+  },
+  "treatments": {
+    "description": "Parameters related to specific treatments",
     "value": {
       "emb": {
-        "dominant": 0.95,
-        "incompletely_dominant": 0.85,
-        "recessive": 0.05
+        "description": "Emamectin Benzoate",
+        "value": {
+          "f_meanEMB": {
```

Comment:
We are not using Aldrin's treatment constants anymore. These can be removed. My bad, I should have deleted these in my past PR.